server/conf/routes

```diff
@@ -22,3 +22,5 @@ POST    /register                   controllers.UserController.postRegister
 #Login
 GET     /login                      controllers.UserController.getLogin
 POST    /login                      controllers.UserController.postLogin
+GET     /githublogin                controllers.GitHubUser.getLogin
+GET     /githubback                 controllers.GitHubUser.githubCallback(code: String)
```

Comment:
Usually in oauth2:
- redirects to some real host (`devgym.com/githubback`), so you can't intercept it and can't test on localhost
- state (secret) should be unique (but github accepts nonunique? it is strange)